airbyte-cdk/java/airbyte-cdk/typing-deduping/src/testFixtures/kotlin/io/airbyte/integrations/base/destination/typing_deduping/RecordDiffer.kt

```diff
@@ -440,7 +441,16 @@ constructor(
                 Instant.ofEpochMilli(Long.MIN_VALUE)
             } else {
                 try {
-                    Instant.parse(node.asText())
```

Comment:
not a massive fan. Any reason DatabricksDestination can't override this class?

Reply:
Too much plumbing into BaseTypingDedupingTest and lot of copy-paste